### PR TITLE
Ncgames 3

### PR DESCRIPTION
**Pull request by HartThomas**

### CODE REVIEW

**Review by Harrison365 — APPROVED** [review]

File: controller.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const { selectCategories } = require("./model");
+
+exports.getCategories = (req, res, next) => {
+  selectCategories().then((categories) => {
+    console.log(categories);
```
Comment (hevdevs) [inline controller.js:5]: remove console.log before merge

File: db/seeds/run-seed.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const devData = require('../data/development-data/index.js');
 const seed = require('./seed.js');
-const db = require('../connection.js');
+const db = require('../index.js');
 
 const runSeed = () => {
   return seed(devData).then(() => db.end());
```
Comment (hevdevs) [inline db/seeds/run-seed.js:6]: ideally we wanted to be seeding before each test (see the notes on seeding with pg for a refresher!)

File: app.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { getCategories } = require("./controller");
+
+const app = express();
+app.use(express.json());
```
Comment (hevdevs) [inline app.js]: only need app.use(express.json()) one we are making a post or patch request

**Review by hevdevs — APPROVED** [review]
Ignore my comment about the seeding in test I can see that's been changed - just a couple of little bits to do but fine to merge with no further PR

**Pull request merged**